RadarSDK/RadarCoordinate.m

```diff
@@ -78,4 +78,12 @@ - (NSDictionary *)dictionaryValue {
     return @{@"type": @"Point", @"coordinates": @[@(self.coordinate.longitude), @(self.coordinate.latitude)]};
 }
 
++ (NSArray<NSArray *>*)arrayForCoordinates:(NSArray<RadarCoordinate *> *)coordinates {
```

Comment:
We should reconstruct `geometry` exactly, so this may need to be a `GeoJSON` struct instead of just an array of coordinates

Nit but I would also probably move this to `RadarGeofence.m` since I think this helper applies more to geofences than to individual coordinates

Reply:
Makes sense, will do that, and move it over